app/src/main/res/values/dimens.xml

```diff
@@ -93,7 +93,7 @@
     <dimen name="comfortable_min_row_height">44dp</dimen>
     <dimen name="quick_key_size">50dp</dimen>
 
-    <dimen name="yochees_radius_key_corner">5dp</dimen>
+    <dimen name="yochees_radius_key_corner">8dp</dimen>
```

Comment:
This value is arbitrary. If you have strong feelings for this value, then it should be a part of a new theme.

Reply:
I don't have strong feelings for this value, the current keyboard theme makes it impossible to type, if you want to type freely and fast it makes you miss keys. I make the keys larger and since this value makes it feel non natural I enhance it to be larger in order for the keyboard to feel native.